ingestion/src/kvkafka.rs

```diff
@@ -287,7 +287,7 @@ async fn create_timeseries(
     let level = param_get_level(
         level_table.clone(),
         raw_datum.kvid.paramid,
-        raw_datum.kvid.level,
+        raw_datum.kvid.level.unwrap(),
```

Comment:
I think this needs to be `unwrap_or(0)`, since we want to convert `None` to `0` instead of panicking when there's `None`. Also this just converts `None` level, but we should also convert `None` sensor to `0`.

Rather than converting it here, I think it's neater to change the definition of `KvalobsId` on lines 76 and 77 to remove the `Option`, and do the `.unwrap_or(0)` on lines 191 and 192 where we create the `KvalobsId`